collector/supervisord.go

```diff
@@ -47,6 +53,21 @@ func NewSupervisordCollector(logger log.Logger) (Collector, error) {
 		subsystem  = "supervisord"
 		labelNames = []string{"name", "group"}
 	)
+
+	if u, err := url.Parse(*supervisordURL); err == nil && u.Scheme == "unix" {
+		// Fake the URI scheme as http, since net/http.*Transport.roundTrip will complain
+		// about a non-http(s) transports
```

Comment:
Minor nit for the comment.
```suggestion
		// about a non-http(s) transports.
```

Reply:
Updated in next commit push https://github.com/prometheus/node_exporter/pull/1592/commits/b81ec9ee1dd2e5bfa69df21930d540b5fae3674c